spec.html

```diff
@@ -18,6 +18,102 @@ <h1>Array Objects</h1>
         <emu-clause id="sec-properties-of-the-array-prototype-object">
             <h1>Properties of the Array Prototype Object</h1>
 
+            <emu-clause id="sec-array.prototype.sort" oldids="sec-sortcompare">
```

Comment:
`Array.prototype.sort` is updated to move most of the body of the `_SortCompare_` inline abstract closure into a new AO that can be shared to `toSorted`.